pkg/backend/display/diff_test.go

```diff
@@ -137,3 +137,63 @@ func TestDiffEvents(t *testing.T) {
 		})
 	}
 }
+
+func TestHasPolicyViolations(t *testing.T) {
```

Comment:
Consider renaming to TestHasMandatoryPolicyViolations to match the function being tested.